### PR TITLE
Rspec

### PR DESCRIPTION
**Pull request by Ahmed-Benj**
I have finished working on testing logic  methods of the game :
 

Testing methods of classes : 

- Game
- Board

Kindly review my work.

Regards,
Ahmed

### CODE REVIEW

File: spec/tic_tac_toe_spec.rb [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+   |   |   |\n+---+---+---+\n|   |   |   |\n+---+---+---+\n|   |\
+   |   |\n+---+---+---+\n\n")
+    end
+  end
+
+  # DRAW_board board = ['X','O','O','O','X','X','X','X','O']
+  # WIN_board_X board = ['X','O',' ',' ','X','O',' ',' ','X']
+  # WIN_board_X board = ['O','O','X',' ','X','O','X',' ',' ']
+  # WIN_board_O board = [' ',' ','X','O','O','O',' ',' ','X']
+  # NoWIN_NoDRAW_board_X board = [' ',' ',' ','X',' ',' ',' ',' ',' ']
```
Comment (carlos-ssh) [inline spec/tic_tac_toe_spec.rb]: - Remove lines of commented code in your files.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
_Hi @Ahmed-Benj_  👋🏽👋🏽👋🏽
**Congratulations! your project looks very nice.** 🥷🏽👾👨🏽‍💻

Your tests are passing, nice examples. No offenses with your code. The live version is very useful, your instructions in the README file were very helpful following the steps I can run the project locally.

‼️ **Few changes requested** 📝:
- Remove commented code from your files.
- In your README file would be great to have an example for your draw game. (Examples of winning boards).
- Nice to have negative test examples.

These changes going to let you professionalism to your project.

_**Happy Coding!**_ 👾
@Carlos-SSH

File: spec/tic_tac_toe_spec.rb (hunk 1 above)
Comment (Ahmed-Benj) [reply spec/tic_tac_toe_spec.rb]: Done

**Review by george-shammar — APPROVED** [review]
### **STATUS:APPROVED** :heavy_check_mark: :heavy_check_mark: :+1: 

Thank you for making the required changes by removing the commented codes and updating the README. 

[OPTIONAL]

- [ ] Improve your test coverage by testing all your public methods. As long as they are doing something, you want to be sure that they are working properly. I could see some untested methods.

I wish you goodluck with your future projects.

Cheers :smile:

**Pull request merged**